omod/src/main/webapp/fragments/issueDrugAccountList.gsp

```diff
@@ -42,13 +42,119 @@
             "issueId": id
         });
     }
+    jq(function () {
+        jQuery('.date-pick').datepicker({minDate: '-100y', dateFormat: 'dd/mm/yy'});
+        getAccountList();
+
+          //action when the searchField change occurs
+        jq("#fromDate-display").on("change", function () {
+            reloadmyList();
+        });
+
+        jq("#toDate-display").on("change", function () {
+            reloadmyList();
+        });
+
+        jq('#issueName').on("keyup",function(){
+            reloadmyList();
+        });
 
+          //action when the searchField blur occurs
+        jq(".searchFieldBlur").on("blur", function () {
+            reloadmyList();
```

Comment:
reconsider variable and function naming  -  This might help though not related completely:=http://stackoverflow.com/questions/1866794/naming-classes-how-to-avoid-calling-everything-a-whatevermanager